sdk/src/oauth/state-machines/debug-oauth-2026-07-28.ts

```
@@ -0,0 +1,2378 @@
+            } catch (error) {
+              throw new Error(
+                `Failed to request MCP server: ${error instanceof Error ? error.message : String(error)}`
+              );
+            }
+            break;
+
+          case "received_401_unauthorized":
+            // Step 3: Extract resource metadata URL and prepare request
+            const challengeParams = parseBearerAuthenticateParameters(
+              state.wwwAuthenticateHeader,
+            );
```

Biome (2.5.3)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)